src/components/WorksiteFilters.vue

```diff
@@ -264,8 +264,9 @@
                       <div>
                         <div>
                           <div
-                            v-for="option in field.values"
-                            v-if="Boolean(option.value)"
+                            v-for="option in field.values.filter(
+                              (option) => !!option.value,
```

Comment:
Wyy `!!` instead of `Boolean`?

Reply:
@tabiodun thanks for your comment
First of all, I was not sure what type of `option.value` is. I check the next code snippets and it was sth like string, I thought.
Based on this input, I think in order to convert string value into boolean, I simply used `!!` which will get true/false value from the strings.
Is `Boolean()` different from the above intention?